sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/CompactBlockDb.kt

```diff
@@ -42,6 +43,18 @@ interface CompactBlockDao {
     @Insert(onConflict = OnConflictStrategy.REPLACE)
     suspend fun insert(block: List<CompactBlockEntity>)
 
+    @Transaction
```

Comment:
Although we aren't using bulk insert anymore, using a SQLite transaction should prevent this approach from adding any additional overhead.